shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/Shiva.scala

```diff
@@ -60,6 +61,10 @@ class Shiva(val parent: Configurable)
       override def configNamespace: String = "shivavariantcalling"
       override def configPath: List[String] = super.configPath ::: "multisample" :: Nil
       genders = samples.map { case (sampleName, sample) => sampleName -> sample.gender }
+      if (isSomaticVariantCallingConfigured()) {
```

Comment:
I think it won't harm to load this always, no need to changed it when you don't have time ;)